cmd/graph.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+)
+
+// analyzeDepsCmd represents the analyzeDeps command
+var graphCmd = &cobra.Command{
+	Use:   "graph",
+	Short: "Generate a .dot file to be used with Graphviz's dot command.",
+	Long: `A graph.dot file will be generated which can be used with Graphviz's dot command.
+	For example to generate a svg image use:
+	twopi -Tsvg -o dag.svg graph.dot`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		depGraph, deps, _ := getDepInfo()
+		fileContents := "digraph {\ngraph [rankdir=TB, overlap=false];\n"
+		for _, dep := range deps {
+			_, ok := depGraph[dep]
+			if !ok {
+				continue
+			}
+			for _, neighbour := range depGraph[dep] {
+				fileContents += ("\"" + dep + "\" -> \"" + neighbour + "\"\n")
```

Comment:
what do you think about using fmt.Sprintf here rather than string concatenation?